react-app/src/App.js

```diff
@@ -1,23 +1,27 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 import Home from './components/Home';
 import Extract from './components/Extract';
+import ResultPage from './components/ResultPage';
 
-import 'bootstrap/dist/css/bootstrap.min.css';
-import './App.css';
-import './stylesheets/Page.css';
+// import 'bootstrap/dist/css/bootstrap.min.css';
+import './stylesheets/custom.scss';
+import './stylesheets/Page.scss';
 
 function App() {
+  const [extractedData, setExtractedData] = useState(['test', 'test', 'test']);
```

Comment:
```suggestion
  const [extractedData, setExtractedData] = useState([]);
```

Can we have the default here just be empty array? It's later populated via actual extraction so it shouldn't hurt anything